tests/acceptance/features/apiSpacesShares/shareUploadTUS.feature

```diff
@@ -402,3 +402,50 @@ Feature: upload resources on share using TUS protocol
       | Tus-Resumable   | 1.0.0                     |
     Then the HTTP status code should be "460"
     And for user "Alice" the content of the file "/textFile.txt" of the space "Personal" should be "original content"
+
+  @issue-10649
+  Scenario: public uploads a zero byte file to a public share folder
```

Comment:
```suggestion
  Scenario: public uploads a zero byte file to a public share folder
```

Since existing test uses the TUS library and is unable to get and check the response.
So, there is no check for the `location` header on TUS upload.